lib/firefox/webdriver/index.js

```diff
@@ -11,10 +11,37 @@ const get = require('lodash.get');
 const geckodriver = require('@sitespeed.io/geckodriver');
 const defaultFirefoxPreferences = require('./firefoxPreferences');
 
+let hasConfiguredGeckoDriverService = false;
+
 module.exports.configureBuilder = function(builder, baseDir, options) {
   const firefoxConfig = options.firefox || {};
   const moduleRootPath = path.resolve(__dirname, '..', '..', '..');
 
+  if (!hasConfiguredGeckoDriverService) {
+    const geckodriverPath = get(
+      firefoxConfig,
+      'geckodriverPath',
+      geckodriver.binPath()
+    );
+
+    let serviceBuilder = new firefox.ServiceBuilder(geckodriverPath);
+    if (options.verbose >= 2) {
+      // This echoes the output from geckodriver to the console.
+      serviceBuilder.setStdio('inherit');
+      // TODO: serviceBuilder.loggingTo(`${baseDir}/geckodriver.log`);
+      if (options.verbose >= 3) serviceBuilder.enableVerboseLogging();
```

Comment:
Nit:  braces for this if statement

Reply:
I've done this.  The reason for the style is this code was culted from the `chromedriver` implementation.